### DISCUSSION TITLE
How can I manually remove the spinning loading circle

### DISCUSSION THREAD

**Discussion by eshsdhsd** [FreeTubeApp/FreeTube#5159] [General]
I dont want it and how can I manually turn it off as there is no official feature to disable it

**Comment by kommunarr** [comment]
**Note:** We're not obligated to provide support to forks of FreeTube, but this is just as a kindness since we were discussing it earlier.

This will change after our video player migration is merged, but you can just look for `.vjs-loading-spinner` in the `videoJS.css` and set `display: none !important;`.

**Reply by eshsdhsd** [reply]
Thanks man. It's gone
